cmake/bintools/arcmwdt/target.cmake

```
@@ -22,17 +22,8 @@ find_program(CMAKE_GDB     ${CROSS_COMPILE}mdb     PATHS ${TOOLCHAIN_HOME} NO_DE
 
 # MWDT binutils don't support required features like section renaming, so we
 # temporarily had to use GNU objcopy instead
-find_program(CMAKE_OBJCOPY arc-elf32-objcopy)
-if (NOT CMAKE_OBJCOPY)
-  find_program(CMAKE_OBJCOPY arc-linux-objcopy)
-endif()
-
-if (NOT CMAKE_OBJCOPY)
-  find_program(CMAKE_OBJCOPY objcopy)
-endif()
-
-if(NOT CMAKE_OBJCOPY)
-  message(FATAL_ERROR "Zephyr unable to find any GNU objcopy (ARC or host one)")
-endif()
+set(CMAKE_OBJCOPY ${ARCMWDT_HELPER_OBJCOPY})
```

a hard `set()` here is not good.

The use of `find_program()` actually allows users to point to a specific installation of objcopy if they have such need.

Combined with the comment on Zephyr SDK lookup then I propose the following adjustment:
```suggestion
find_program(CMAKE_OBJCOPY ${ZEPHYR_CROSS_COMPILE}objcopy PATHS ${ZEPHYR_SDK_INSTALL_DIR} NO_DEFAULT_PATH)
```